lib/src/models/issue_wizard.dart

```diff
@@ -173,6 +173,7 @@ class IssueWizardItem {
   @JsonKey(name: 'inapp', defaultValue: false)
   final bool inApp;
 
+  @JsonKey(name: 'completed', defaultValue: false)
   final bool completed;
```

Comment:
I merged my PR already where all these `defaultValue` things are superfluous when they are exactly the same as the default value being defined in the constructor. When resolving the merge conflict, you might have to deal with this.